mrp_product_document_link/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Manufacture Product Document Link",
+    "summary": """
+    Creates a shortcut to the product documents in the product form view from mrp.
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
```

Comment:
Wrong license (as `document_url` is `AGPL`):
```suggestion
    "license": "AGPL-3",
```